src/track/playcounter.cpp

```diff
@@ -0,0 +1,22 @@
+#include "track/playcounter.h"
+
+
+void PlayCounter::updatePlayed(bool bPlayed) {
+    if (bPlayed) {
+        // increment the play count when play during the current session
+        ++m_iTimesPlayed;
+        m_bPlayed = true;
+    } else if (m_bPlayed) {
+        // decrement the play count, but only if already played during
+        // the current session
+        if (0 < m_iTimesPlayed) {
```

Comment:
This condition should never be false. 
If it is false we had an inconsistent state before. Played checked but play counter at 0. 
Lets fix this instead, if that still can happen.